### PR TITLE
Fixes: git-find-dirs-many-files.sh and git-find-lfs-extensions.py

### PR DESCRIPTION
**Pull request by lavinia8**
- Changed to numeric sort ( `sort -k 1 -n -r` ); 
- Changed to counting the direct members (=files) only ( `find "$0" -maxdepth 1 -type f` );
   - Is there any reason not to count the direct members only, but counting the members recursively?
e.g. Considering `subdir1` has 100 direct members (=files) and the rest of parent folders have no direct members (=files), this is what the script identifies without my fix:
   ```
   100 .\dir\subdir0\subdir1
   100 .\dir\subdir0
   100 .\dir
   100 .\
   ```
   This is what the script identifies after my fix:
   ```
   100 .\dir\subdir0\subdir1
   0 .\dir\subdir0
   0 .\dir
   0 .\
   ```

### CODE REVIEW

**Comment by lavinia8** [comment]
git-find-lfs-extensions.py: changed `if len(sys.argv):` to `if len(sys.argv)>1:`
- sys.argv[0] = script itself; sys.argv[1] = size;

File: git-find-lfs-extensions.py [hunk 1]
```diff
@@ -26,7 +26,7 @@
 import sys
 
 # Threshold that defines a large file
-if len(sys.argv):
+if len(sys.argv)>1:
```
Comment (larsxschneider) [inline git-find-lfs-extensions.py:29]: 👍 I would like to merge this

File: git-find-dirs-many-files.sh [hunk 2]
```diff
@@ -20,7 +20,7 @@ else
 fi
 
 IFS=$'\n';
-DIRS=$(find . -type d -not -path "./.git/*" -exec bash -c 'COUNT=$(find "$0" -type f | wc -l); echo "$COUNT $0"' {} \; | sort -r)
+DIRS=$(find . -type d -not -path "./.git/*" -exec bash -c 'COUNT=$(find "$0" -maxdepth 1 -type f | wc -l); echo "$COUNT $0"' {} \; | sort -k 1 -n -r)
```
Comment (larsxschneider) [inline git-find-dirs-many-files.sh:23]: > - Changed to counting the direct members (=files) only ( `find "$0" -maxdepth 1 -type f` );
>    - Is there any reason not to count the direct members only, but counting the members recursively?
> e.g. Considering `subdir1` has 100 direct members (=files) and the rest of parent folders have no direct members (=files), this is what the script identifies without my fix:
>    ```
>    100 .\dir\subdir0\subdir1
>    100 .\dir\subdir0
>    100 .\dir
>    100 .\
>    ```
>    This is what the script identifies after my fix:
>    ```
>    100 .\dir\subdir0\subdir1
>    0 .\dir\subdir0
>    0 .\dir
>    0 .\
>    ```
This is not my use case here. I want to find entire directory trees that I can exclude from a Git repo and therefore I need to count recursively. Does this make sense?

Comment (larsxschneider) [reply git-find-dirs-many-files.sh:23]: @lavinia8 ping 😄

**Pull request closed without merging**